edgebit/agent/v1alpha/inventory_service.proto

```diff
@@ -34,10 +37,28 @@ message Container {
     string name = 1;
 }
 
+// namespaces
+// k8s, vm, ec2, ec2label, gce, user-label:bu=accounting
+
+enum LabelNamespace {
+    LABEL_NAMESPACE_UNSPECIFIED = 0;
+    LABEL_NAMESPACE_DOCKER = 1;
+    LABEL_NAMESPACE_K8S = 2;
+    LABEL_NAMESPACE_VM = 3;
+    LABEL_NAMESPACE_EC2 = 4;
+    LABEL_NAMESPACE_GCE = 5;
+    LABEL_NAMESPACE_AZURE = 6;
+    LABEL_NAMESPACE_USER = 7;
+}
+
+message Label {
+    LabelNamespace ns = 1;
+    string name = 2;
+    string value = 3;
+}
+
 message Workload {
-    // Group path to indicate where the workload fits in organizationally.bool
-    // Example: [ "host-a23cd4", "k8s", "pod-nginx" ]
-    repeated string group = 1;
+    repeated Label labels = 1;
```

Comment:
This is technically a breaking change but b/c the field was never used, it should not break anything.